src/types.ts

```diff
@@ -95,3 +96,79 @@ export enum Flavor {
   Elasticsearch = 'elasticsearch',
   OpenSearch = 'opensearch',
 }
+
+// TODO add raw data and logs
+export enum LuceneQueryType {
+  Traces = 'Traces',
+  Metric = 'Metric',
+}
+
+export type LuceneQueryObj = {
+  size: number;
+  query: DSLQuery;
+  // TODO we could type this further, default trace queries are very hard coded right now and do not change much.
```

Comment:
If we merge this we should create a follow up issue in the backlog for this

Reply:
good catch! just updated so we don't have this unknown anymore